Enable log trace context injection by default (#44)
DataDog/datadog-lambda-layer-python: README.md

- DD_FLUSH_TO_LOG

> To connect logs and traces, set the environment variable below to `True`. The default format of the AWS provided `LambdaLoggerHandler` will be overridden to inject `dd.trace_id` and `dd.span_id`. The default Datadog lambda log integration pipeline will automatically parse them and map the `dd.trace_id` into the reserved [trace_id attribute](https://docs.datadoghq.com/logs/processing/#trace-id-attribute). Defaults to true.

Reviewer: ```suggestion
Inject Datadog trace id into logs for correlation. Defaults to true.
```

If your Lambda function is deployed by the Serverless Framework, such a mapping template gets created by default.

## Log and Trace Correlations

> By default, the Datadog trace id gets automatically injected into the logs for correlation, if using the AWS provided `LambdaLoggerHandler`.

Reviewer: ```suggestion
By default, the Datadog trace id gets automatically injected into the logs for correlation, if using the standard python `logging` library.
```

> If you use a custom logger handler to log in json, you can manually inject the ids using the helper function `get_correlation_ids`.

Reviewer: ```suggestion
If you use a custom logger handler to log in json, you can inject the ids using the helper function `get_correlation_ids` [manually](https://docs.datadoghq.com/tracing/connect_logs_and_traces/?tab=python#manual-trace-id-injection).
```
